governance/pyth_staking_sdk/src/pyth-staking-client.ts

```diff
@@ -810,17 +826,43 @@ export class PythStakingClient {
       .remainingAccounts(
         remainingAccount
           ? [
-              {
-                pubkey: remainingAccount,
-                isWritable: false,
-                isSigner: false,
-              },
-            ]
+            {
+              pubkey: remainingAccount,
+              isWritable: false,
+              isSigner: false,
+            },
+          ]
           : [],
       )
       .instruction();
   }
 
+  public async getJoinDaoLlcInstructionIfNeeded(
```

Comment:
I would make these separated functions.

`hadJoinedDaoLlc` and `getJoinDaoLlcInstruction`

Then you dont have to check if this the result of this function is undefined

Reply:
cool